### ISSUE TITLE
[FEAT] Move `@daffodil/cart` checkout code to `@daffodil/checkout`

### ISSUE THREAD

**Issue by griest024** [graycoreio/daffodil#2807]
# :bulb: Feature request

## Feature Name

Move `@daffodil/cart` checkout code to `@daffodil/checkout`

## The Desired Behavior

Some fields on cart are really more part of checkout and don't really belong on cart. `DaffCart` should be a collection of cart items and some totals and discounts. Shipping and billing address, shipping method, and payment are not needed for that purpose and should be moved to checkout.

## Your Use Case

As an app dev, I don't want to load unnecessary code on my homepage when I load the cart module.

## Tasks
- Move code to checkout
- Deprecate moved code in cart
- Change cart glue packages to depend on checkout instead

**Comment by griest024** [comment]
blocked by #2751
